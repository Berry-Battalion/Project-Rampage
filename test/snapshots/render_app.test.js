import React from 'react';
import App from '../../src/components/app';
import renderer from 'react-test-renderer';

describe('<App />', () => {
    test('renders correctly', () => {
        const tree = renderer.create(
            <App />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
})