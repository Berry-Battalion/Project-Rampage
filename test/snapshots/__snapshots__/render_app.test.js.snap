// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div>
  <div>
    Header
  </div>
  <div>
    WRONG!
  </div>
</div>
`;
